src/samplers/cpu/macos/usage/mod.rs

```diff
@@ -124,22 +124,26 @@ impl CpuUsage {
                     .offset((cpu as i32 * libc::CPU_STATE_MAX + libc::CPU_STATE_NICE) as isize)
                     as u64)
                     .wrapping_mul(self.nanos_per_tick);
+                let busy = user.wrapping_add(system.wrapping_add(idle));
```

Comment:
Shouldn't busy be user + system + nice, not user + system + idle?

Reply:
Good catch. Fixed.